keyboards/takashicompany/rookey/info.json

```diff
@@ -0,0 +1,47 @@
+{
+    "manufacturer": "takashicompany",
+    "keyboard_name": "Rookey",
+    "maintainer": "takashicompany",
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "B4", "pin_b": "E6"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "direct": [
+            ["F4", "F5", "F6", "F7", "B1", "B3", "B2", "B6", "B5"]
+        ]
+    },
+    "url": "https://github.com/takashicompany/rookey",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0056",
+        "vid": "0x7463"
+    },
```

Comment:
```suggestion
    },
    "dynamic_keymap": {
        "layer_count": 32
    },
```

Reply:
I don't know. 16 is more reasonable.  And 8 would be more than plenty.